omero/sysadmins/repository-move.txt

```diff
@@ -117,3 +117,37 @@ to ``/Volumes/imports/ManagedRepository``:
     If :property:`omero.managed.dir` is set then the Managed Repository and
     the OMERO data directory should be treated independently and thus be moved
     separately if necessary.
+
+Extending the Managed Repository
+--------------------------------
+
+It is possible to leave the Managed Repository in place yet have newly
+imported image files stored on a different underlying storage volume.
+For example, if your :property:`omero.managed.dir` is set to
+:file:`/mnt/omero/ManagedRepository` and, as it fills, it would be
+better for future images to be imported onto
+:file:`/mnt/omero/large-volume-B/` then an OMERO administrator may use
+the :omerocmd:`fs mkdir` subcommand to properly set up a subdirectory in
+the existing Managed Repository:
+::
+
+    bin/omero fs mkdir volume-B
```

Comment:
However, this is a pretty minor comment, since by re-reading I worked it all out (as any user would do), so if it's clearer to keep it like it is then that's OK with me.